.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+# Contributing to pyccel
+
+We welcome any and all contributions to pyccel!
+
+The following is a summary of the best ways to interact/help with this project.
+
+## Asking questions
+
+For simple questions (e.g. installation problems), we encourage you to use the [discussions tab](https://github.com/pyccel/pyccel/discussions/categories/q-a).
+
+For more involved questions we also have a [slack channel](pyccel.slack.com).
+
+## Requesting features
+
+To request a new feature, please use the [discussions tab](https://github.com/pyccel/pyccel/discussions/categories/ideas).
+This allows us to discuss the best implementation in the target languages before creating an issue to work on.
+
+## Reporting issues
+
+When reporting issues please include as much detail as possible about your
+operating system, and python version. Whenever possible, please
+also include a brief, self-contained code example that demonstrates the problem.
+
+## Ways to help
+
+There are many ways to help with the pyccel project which are more or less involved.
+
+### Answering questions
+
+Questions are sometimes asked in the [discussions tab](https://github.com/pyccel/pyccel/discussions/categories/q-a) or in issues.
+Any help answering these questions is appreciated as it helps our users find solutions as fast as possible.
+
+### Documentation
+
+Reviewing the existing documentation is a great way to help out!
+We need to ensure that it is up-to-date and relevant, and that the examples are clear, relevant, and work correctly.
+
+Missing documentation can also be added to improve the existing docs.
+
+Please open a pull request to add or change anything in the docs.
+
+### Reviewing a pull request
+
+There are many pull requests and reviewing them is a great way to help things get to the master branch faster.
+This is also a really good way to get to grips with the code base as you will see examples of many different areas of the code.
+It's incredibly helpful to have pull requests go through as many reviews as possible, to make sure the code change makes sense, is documented, and is efficient and clear.
+As the clarity is subjective, more eyes can only improve the code base.
+The review process is described in the [developer docs](https://github.com/pyccel/pyccel/blob/master/developer_docs/review_process.md), keep an eye out for PRs tagged `needs_initial_review`.
+
+### Contributing code
+
+Firstly, thanks for your interest in contributing code to pyccel!
+
```

Comment:
This message is very motivating, I think it will be great if we move it to the top.

Reply:
Fixed [a971b8b](https://github.com/pyccel/pyccel/pull/1252/commits/a971b8b5b296253d5a0fe2d5155fbf75b117fdaa)